### ISSUE TITLE
Error: Null check operator used on a null value

### ISSUE THREAD

**Issue by bradsbrown** [videah/SkyBridge#53]
I've been loving Skybridge as a user of the primary instance, and wanted to try to get my own set up and running. However, every attempt to build fails with an error:
```
An exception occurred while executing hook: {$HOME}/.mason-cache/bundled/dart_frog_prod_server_0.1.0+1_dd63917756d9b4f8fd323148a45cb9d6c977ffb9/hooks/pre_gen.dart.
Error: Null check operator used on a null value
```

I'm mostly a Python guy, so figuring out Dart enough to solve this has been a bit of a battle. I've gotten as far as to get a sense that `dart-sdk` v3 was released earlier this year and it strongly enforces some rules around null checks, and I've found a few places in `routes/` where null checks appear to be used, so it's possible that's where the error is coming from. But that error message doesn't give me any detail on _where_ the issue was found, so I'm a bit stuck on what to fix or how to fix it so that I can get this to build. It looks like [ci has been failing](https://github.com/videah/SkyBridge/actions) for a bit for the same reason, as well.

Happy to contribute a fix if someone with a bit more experience can point me in the right direction.

**Comment by videah** [comment]
This is an issue stemming from one of `dart_frog`'s dependencies as far as I know, I wasn't able to narrow it down but it only happens when building in Docker. It might be worth opening an issue on their repo, I'll see if I can find more time to further narrow it down.

**Comment by bradsbrown** [comment]
Hrm, interesting. I was hitting the same error running a `dart_frog [dev|build]` locally outside of docker as well.

**Comment by videah** [comment]
Temporarily fixes as of c58a45c
